docs/user-guide/structured.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Suggest adding the grid lines, or coastlines for the data to better visualize the singularity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1040/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>